Extensions/old_notifications.js

```diff
@@ -121,23 +124,23 @@ XKit.extensions.old_notifications = new Object({
 			}
 		}
 
-		var is_friend = $(obj).hasClass("is_friend");
-		var is_reblog = $(obj).hasClass("is_reblog");
-		var is_reply = $(obj).hasClass("is_reply");
-		var is_answer = $(obj).hasClass("is_answer");
-		var is_like = $(obj).hasClass("is_like");
-		var is_follow = $(obj).hasClass("is_follower");
-		var is_mention = $(obj).hasClass("is_user_mention");
+		var is_friend = obj.hasClass("is_friend");
+		var is_reblog = obj.hasClass("is_reblog");
+		var is_reply = obj.hasClass("is_reply");
+		var is_answer = obj.hasClass("is_answer");
+		var is_like = obj.hasClass("is_like");
+		var is_follow = obj.hasClass("is_follower");
+		var is_mention = obj.hasClass("is_user_mention");
```

Comment:
There's also a new type, `note_mention`, for when someone @-mentions you in a *reply* to a post. I would suggest including that in the same selector.